app/commands/user/add_roles.rb

```diff
@@ -0,0 +1,10 @@
+class User::AddRoles
+  include Mandate
+
+  initialize_with :user, :roles
```

Comment:
I considered making roles a splat argument, but this version is a bit more "obvious." Let me know what you think.